test/e2e-common.sh

```diff
@@ -162,6 +162,42 @@ function gateway_conformance() {
   popd
 }
 
+function test_conformance_no_gw_service() {
+  local parallel_count="12"
+  if (( KIND )); then
+    parallel_count="1"
+  fi
+
+  echo "Setting custom config-gateway"
+  custom_configs_dir="${REPO_ROOT_DIR}/test/modified-config-gateway"
+  if [[ "${INGRESS}" == "contour" ]]; then
+    kubectl apply -f "${custom_configs_dir}/contour-no-service-vis.yaml" -n "${SYSTEM_NAMESPACE}"
+    echo "Waiting 30s for change to get picked up."
+    sleep 30
+  else
+    kubectl apply -f "${custom_configs_dir}/istio-no-service-vis.yaml" -n "${SYSTEM_NAMESPACE}"
+    echo "Waiting 30s for change to get picked up."
+    sleep 30
+  fi
+
+  go_test_e2e -timeout=20m -tags=e2e -parallel="${parallel_count}" ./test/conformance \
```

Comment:
We're running conformance again `./test/conformance` - I'm guessing we want to run `./test/e2e/`